webpack/assets/javascripts/react_app/components/passwordStrength/password_strength.scss

```diff
@@ -0,0 +1,10 @@
+.ReactPasswordStrength-input {
+  width: 100% !important;
```

Comment:
Can we style this without the `!important`s?  Maybe add additional classes or elements to the selector if you need more specificity.